lib/assertions.js

```diff
@@ -135,8 +136,11 @@ const oidc = {
       nonce,
       accessToken = crypto.randomBytes(15).toString('hex'),
     ) => {
-      const sub = `s=${nric},u=${uuid}`
-
+      if(nric.startWith('Y')){
+        const sub = `s=${nric},fid='G730Z-H5P96',coi='DE',u=${uuid}`                
+      }else{ 
+        const sub = `s=${nric},u=${uuid}`
+      }
```

Comment:
Typo - The method name should be `startsWith()`, not `startWith()`. Also, please observe linting norms for the codebase.